README.md

```diff
@@ -1,14 +1,15 @@
 # mysql-backup-restore
-Service to backup and/or restore mysql databases to/from S3
+Service to backup and/or restore mysql databases to/from S3 and optionally to B2
 
 ## How to use it
 1. Create an S3 bucket to hold your backups
 2. Turn versioning on for that bucket
+2. (Optional) Create a B2 bucket to hold your backups
 3. Supply all appropriate environment variables
-4. Run a backup and check your bucket for that backup
+4. Run a backup and check your bucket(s) for that backup
 
 ### Environment variables
-`MODE` Valid values: `backup`, `restore`
+`MODE` Valid values: `backup`, `restore`. Restores are implemented **only** from S3.
```

Comment:
Am I reading this correctly, that you can backup to B2 but you cannot restore from B2?

Reply:
Correct.  The intent was to have a backup copy that was **not** part of the AWS infrastructure or account. If someone was to destroy the system on AWS (including the backups on AWS S3), you'd still have the backups on Backblaze B2.  Rebuilding the AWS infrastructure would give you an empty S3 bucket and you could copy the database backup file from B2 to S3 and restore from S3 as normal.